### PR TITLE
Creating an example for Azure Queues

### PR DESCRIPTION
**Pull request by tamarastanojevic**
Creating an example project for adding custom spans for Azure Queues

### CODE REVIEW

File: azure-queues-example/azure-queues-example/DequeueManager.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                {
+                    HandleMessage(queueMessage, queue);
+                }
+            }
+        }
+
+        private static DistributedTraceInformation ExtractCorrelationData(string messageBody)
+        {
+            string[] messageParts = messageBody.Split('|');
+            string[] correlationParts1 = messageParts[1].Split('=');
```
Comment (zivaninstana) [inline azure-queues-example/azure-queues-example/DequeueManager.cs]: We should use the last two parts always. So, maybe messageParts[messageParts.Length - 1] and messageParts[messageParts.Length - 1] is a better example.

File: azure-queues-example/azure-queues-example/DequeueManager.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            DistributedTraceInformation info = new DistributedTraceInformation();
+            info.ParentSpanId = SpanIdUtil.FromExternalSpanId(correlationParts1[1]);
+            info.TraceId = SpanIdUtil.FromExternalSpanId(correlationParts2[1]);
+
+            return info;
+        }
+
+        private string ExtractMessageText(BinaryData messageBody)
+        {
+            string[] messageParts = messageBody.ToString().Split('|');
```
Comment (zivaninstana) [inline azure-queues-example/azure-queues-example/DequeueManager.cs:41]: Same here

**Review by zivaninstana — CHANGES_REQUESTED** [review]
Looks good except few things 👍

File: azure-queues-example/azure-queues-example/DequeueManager.cs (hunk 2 above)
Comment (tamarastanojevic) [reply azure-queues-example/azure-queues-example/DequeueManager.cs:41]: Done

File: azure-queues-example/azure-queues-example/DequeueManager.cs (hunk 1 above)
Comment (tamarastanojevic) [reply azure-queues-example/azure-queues-example/DequeueManager.cs]: Done

**Pull request merged**